### ISSUE TITLE
Panel chime switch entity disappears in V.0.9.0 (DSC)

### ISSUE THREAD

**Issue by aslm1** [ufodone/envisalink_new#113]
I'm using a DSC 1832 Panel with an Envisalink 4MAX.  Firmware: 01.00.44.

When I upgrade from V.0.8.2 to V0.9.0 the panel chime switch entity shows 'This entity is no longer being provided by the envisalink_new integration. If the entity is no longer in use, delete it in settings.'

If I downgrade to V.0.8.2 the switch entity works as expected.

**Comment by ufodone** [comment]
My own chime switch is showing up on 0.9.0/DSC so my guess would be that your config entry may be incorrectly set to a different panel type (e.g., Honeywell or simply `null`).  I've seen a few cases where this happens to people but I haven't yet figured out how it is happening.

If you're able, can you look at the `envisalink_new` entry in the `.storage/core.config_entries` file located in HA's config directory (where configuration.yaml is)?  It would look something like:

```
      {
        "entry_id": "461c2053f29eb6a84120e8699cc975cf",
        "version": 1,
        "minor_version": 1,
        "domain": "envisalink_new",
        "title": "Home Alarm",
        "data": {
          "port": 4025,
          "evl_version": 4,
          "panel_type": "DSC",
...
```

The `panel_type` field should be set to `DSC` like this example.

**Comment by aslm1** [comment]
Here is what I believe to be the relevant section.  `panel_type` is in fact showing as `null`.


```       {
        "entry_id": "728450a69974d57e7b7a3ff64c893107",
        "version": 1,
        "minor_version": 1,
        "domain": "envisalink_new",
        "title": "Alarm",
        "data": {
          "host": "XXX.XXX.XXX.XXX",
          "user_name": "XXXXXX",
          "password": "XXXXXXX",
          "partition_set": "1",
          "zone_set": "1-6",
          "port": 4025,
          "discovery_port": "80",
          "code": "XXXX",
          "evl_version": "4MAX",
          "panel_type": null
        },
```

**Comment by ufodone** [comment]
I've published a new v0.9.1 release which attempts to detect and correct the null `panel_type` entry.  If you haven't already manually corrected your entry, please let me know if this release fixes your issues.

**Comment by aslm1** [comment]
Unfortunately, I already manually corrected the entry (which resolved the chime switch issue).

I have upgraded to v0.9.1 and it is still working.
